packages/guides/src/Nodes/Metadata/OrphanNode.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of phpDocumentor.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @link https://phpdoc.org
+ */
+
+namespace phpDocumentor\Guides\Nodes\Metadata;
+
+/**
+ * If set, warnings about this file not being included in any toctree will be suppressed.
+ */
+class OrphanNode extends MetadataNode
+{
+    public function __construct()
+    {
+        parent::__construct('true');
```

Comment:
It feels like we need some boolean metadata node? Or should these be "options" on a document?

Reply:
It is really a Metadata Node without content. But as an empty string is usually treated similar to false I just put another String?